src/cljs/main/broadfcui/page/workspace/data/copy_data_workspaces.cljs

```diff
@@ -30,30 +30,30 @@
                   {:header "Created By" :starting-width 200}
                   (table/date-column {})
                   {:header "Access Level" :starting-width 106}
-                  {:header "Protected" :starting-width 86
-                   :content-renderer #(if % "Yes" "No")}]
+                  {:header "Authorization Domain" :starting-width 150
+                   :content-renderer #(if % % "None")}]
```

Comment:
**Before:** Workspace list in data import page shows whether or not the workspace is protected by the dbGapAuthorizedUsers realm
**After:** ... shows which authorization domain the workspace is a part of